kivy/lang.py

```diff
@@ -1795,6 +1795,10 @@ def match(self, widget):
             parents[cls] = classes
         return self.key in parents[cls]
 
+    def match_widget_name(self, widget_name):
+        return self.key == widget_name.lower()
+
+
 
```

Comment:
Did you run the pep8 hook? Because it should be exactly 2 blank lines: https://kivy.org/docs/contribute.html#coding-style.